samples/bluetooth/pbp_public_broadcast_source/prj.conf

```diff
@@ -24,3 +24,4 @@ CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT=1
 
 CONFIG_BT_EXT_ADV=y
 CONFIG_BT_DEVICE_NAME="PBS"
+CONFIG_STATIC_BROADCAST_ID=n
```

Comment:
Why is this sample defaulting to the random ID?

Reply:
No special reason, other than I thought it was nice to have samples using both random and static.

Reply:
Yeah, thought so - That makes sense :)